app/assets/javascripts/components/projects/ProjectBoard.js

```diff
@@ -70,18 +143,22 @@ export class ProjectBoard extends React.Component {
                 state: Story.status.UNSTARTED
               })}
             />}
+          columnId={'backlog'}
```

Comment:
You don't need `{ }` when passing strings through props. You are using this in other props too. Unlesse the project enforces this, you should remove it.

Reply:
Yeah, I'll remove it.